code/src/components/GameBoardFooter.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+
+import { useDispatch } from 'react-redux';
+import { gameReducer } from '../reducers/gameReducer';
+
+export const GameBoardFooter = () => {
+
+  const dispatch = useDispatch();
+  
+  const handleEndGameClick = (event) => {
+    event.preventDefault();
+    dispatch(gameReducer.actions.endGame(true));
+  }
+
+  return(
+    <section className="gameboard-footer">
+      <button 
+        onClick={(event) => handleEndGameClick(event)}
+        className="gameboard-end-button">
+        End Game
+      </button>
+    </section>
```

Comment:
We're not sure if there are any conventions on what to include in the footer? This could also be just a section. If you do decide to have a footer you could change the `section` to be a `footer` instead and include stuff like creator, copyrights and contacts.